scripts/generate_sitemap.py

```diff
@@ -57,6 +58,16 @@ def __init__(self):
         if not os.path.exists(self.sitemap_dir):
             print('Creating sitemap directory at `{}`'.format(self.sitemap_dir))
             os.makedirs(self.sitemap_dir)
+        if settings.SITEMAP_TO_S3:
+            assert settings.SITEMAP_AWS_BUCKET, 'SITEMAP_AWS_BUCKET must be set for sitemap files to be sent to S3'
+            assert settings.AWS_ACCESS_KEY_ID, 'AWS_ACCESS_KEY_ID must be set for sitemap files to be sent to S3'
+            assert settings.AWS_SECRET_ACCESS_KEY, 'AWS_SECRET_ACCESS_KEY must be set for sitemap files to be sent to S3'
+            self.s3 = boto3.resource(
```

Comment:
I'll also note from some local testing that setting the resource doesn't validate anything, so this won't fail until you try to do anything.